src/v2/Apps/Order/Routes/__tests__/Offer.jest.tsx

```diff
@@ -313,12 +326,13 @@ describe("Offer InitialMutation", () => {
       })
     })
 
-    it("tracks viwing the low offer speedbump", async () => {
+    it.skip("tracks viwing the low offer speedbump", async () => {
```

Comment:
ended up the tests here were not updated to the new price options, so there are some inconsistencies and wrong assertions given some mocked data. We should refactor these tests here https://artsyproduct.atlassian.net/browse/NX-3098?atlOrigin=eyJpIjoiNTAxNjZiNjU1YmVmNDE4ZmIxMzQzNzMxYTE3ZWU0MGIiLCJwIjoiamlyYS1zbGFjay1pbnQifQ

Reply:
Thank you for ticketing 🙏